packages/api-reference/src/components/Content/Schema/SchemaDiscriminator.vue

```diff
@@ -64,14 +64,14 @@ const getModelNameFromSchema = (schema: any): string | null => {
     return null
   }
 
-  if (schema.name) {
-    return schema.name
-  }
-
   if (schema.title) {
```

Comment:
good call! prob got change within some previous changes by mistake, def should prioritize human-readable first!

Reply:
can you update also the test in `SchemaDiscriminator.test.ts` to swap precedence?